web/prompt_gallery.js

```diff
@@ -1660,30 +1660,15 @@ class PromptGallery {
         return imgContainer;
     }
 
-    cleanText(text) {
-        // Remove leading and trailing commas, spaces, and BREAK
-        text = text.replace(/^[,\s]+|[,\s]+$/g, '');
-        // Replace BREAK (case insensitive) with a period, handling various scenarios
-        text = text.replace(/\s*BREAK\s*(?:,\s*)?/gi, '. ');
-        // Remove any duplicate periods or comma-period combinations
-        text = text.replace(/\.{2,}/g, '.').replace(/,\s*\./g, '.');
-        // Ensure there's a space after each period or comma, but not at the very end
-        text = text.replace(/([.,])(?=\S)/g, '$1 ').trim();
-        return text;
-    }
-
     combineTexts(existing, newText) {
-        existing = this.cleanText(existing);
-        newText = this.cleanText(newText);
-        
         if (!existing) return newText;
         
         // If existing text ends with a period, don't add a comma
         if (existing.endsWith('.')) {
             return existing + ' ' + newText;
-        } else {
-            return existing + ', ' + newText;
         }
+
+        return existing + newText;
```

Comment:
I agree, cleaning the existing text can break the entire prompt. I hope this gets merged :)
If the original issue was to be sure to have a comma separating to new tags from the existing tags, maybe a fix can be:
```suggestion
        return existing.split(",").concat(newText.split(",")).join(",");
```